core/templates.yml

```diff
@@ -376,6 +378,11 @@ paths:
     sequence_asset_publish_cache:           '@sequence_asset_publish_area_anim/alembic/@geom_subdirs/@sequence_asset_version_name.abc'
     shot_asset_publish_cache:               '@shot_asset_publish_area_anim/alembic/@geom_subdirs/@shot_asset_version_name.abc'
 
+    # STANDIN
```

Comment:
I know this only makes sense in an asset context, but can we add shot/seq/project templates as well for completeness?
I notice we have not configured any apps to use these templates yet. Once you do that, some apps may search for the templates (even in shot context) and error out saying template not available. I'm not completely sure exactly when that happens, but seen it a lot in ingest.